modules/statistics/statistics.js

```diff
@@ -710,16 +706,26 @@ Statistics.reportGlobalError = function(error) {
 };
 
 /**
- * Sends event to analytics and callstats.
- * @param {string} eventName the event name.
- * @param {Object} data the data to be sent.
+ * Sends event to analytics and logs a message to callstats.
+ *
+ * @param {string | Object} eventName the event name, or an object which
+ * represents the entire event.
+ * @param {Object} properties properties to attach to the event
  */
-Statistics.sendEventToAll = function(eventName, data) {
-    this.analytics.sendEvent(eventName, data);
-    Statistics.sendLog(
-        JSON.stringify(
-            {
-                name: eventName,
-                data
-            }));
+Statistics.sendEventAndLog = function(eventName, properties = {}) {
+    let eventToLog = null;
```

Comment:
Well, `eventToLog` will be assigned bellow so I don't much see the point in saying that it's `null`. As if it's possible that it's equal to `null`.

Reply:
Removed the assigment